pootle/settings/40-apps.conf

```diff
@@ -131,6 +131,10 @@ POOTLE_SIGNUP_ENABLED = True
 # http://django-statici18n.readthedocs.io/en/latest/faq.html#using-a-placeholder-directory
 STATICI18N_ROOT = working_path('static')
 
+# Directory where generated files will be written to. Customized to avoid name
+# clashing with `static/js`.
+STATICI18N_OUTPUT_DIR = 'translations'
```

Comment:
there is a real risk of this being confused with the Pootle directory of the same name